developer/src/common/include/kmn_compiler_errors.h

```diff
@@ -36,7 +36,11 @@ namespace CompilerErrorSeverity {
   };
 };
 
-#define MESSAGE_SEVERITY_MASK 0xF00000
+#define MESSAGE_SEVERITY_MASK    0x00F00000  // includes reserved bits, 16 possible severity levels
+#define MESSAGE_ERROR_MASK       0x000FFFFF  // error | namespace
+#define MESSAGE_NAMESPACE_MASK   0x000FF000  // 256 possible namespaces
+#define MESSAGE_BASEERROR_MASK   0x00000FFF  // error code, 2,048 possible error codes per namespace
+#define MESSAGE_RESERVED_MASK    0xFF000000  // do not use these error values at this time
```

Comment:
Checks out with some of the code changes I noticed within #12044; the changes in #12044 that triggered the connection for me all line up well with `MESSAGE_SEVERITY_MASK` there.